presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,68 @@
+* MinbyN/maxbyN's third argument (N) is now checked to be unique.
+* Print information about cost-based optimizers and the source of stats they use (CBO/HBO) in explain plans when session property verbose_optimizer_info_enabled=true.
+* The Iceberg connector now supports `ALTER TABLE <table> ADD COLUMN <column> [WITH (partitioning = '<transform_func>')]` statements to indicate partition transform when add column to table.
+* The Iceberg connector now supports `DELETE FROM <table> [where <filter>]` statements to delete one or more entire partitions.
+* Track null probe keys in join and use it to enable outer join null salt in history based optimization.
+* Update Babel and related npm packages to solve critical vulnerability: https://github.com/advisories/GHSA-67hx-6x53-jw92 The Babel packages are used to generate the JavaScript files for Presto UI. The update prevents the build servers or developers' machines from running arbitrary code while building the JavaScript files.
+* Update Joda-Time to 2.12.5 to use 2023c tzdata. Note: a corresponding update to the Java runtime should also be made to ensure consistent timezone data. For example, Oracle JDK 8u381, tzdata-java-2023c rpm for OpenJDK, or use Timezone Updater Tool to apply 2023c tzdata to existing JVM.
+* Updated docs accordingly (hive connector - schema evolution).
+* When partial aggregation statistics are used in PushPartialAggregationThroughExchange, the optimization also applies to multi-key aggregations (as opposed to single-key only when CBO is used).
+
+Hive Changes
+____________
+* Changes in ParquetPageSourceFactory so that the type-check does not fail with the new widenings supported.
+* Support for Parquet writer versions V1 and V2 is now added for Hive and Iceberg catalogs. It can be toggled using session property `parquet_writer_version` and config property `hive.parquet.writer.version`. Valid values for these properties are PARQUET_1_0 and PARQUET_2_0. Default is PARQUET_2_0. E.g., set session parquet_writer_version=PARQUET_1_0 / hive.parquet.writer.version=PARQUET_1_0.
+
+Iceberg Changes
+_______________
+* Fix iceberg table creation using glue metastore.
+* SHOW STATS command will support timestamp column based on user session's time zone.
+* Support view creation via Iceberg connector.
+* Timestamp with time zone data type is not allowed in create table and alter table statements.
+* Update Iceberg version from 1.3.1 to 1.4.1.
+
+Pinot Changes
+_____________
+* Fix pinot single quote literal push down issue.
+
+Prestissimo (native Execution) Changes
+______________________________________
+* Add support for internal authentication using JWT. It can be configured using configs "internal-communication.jwt.enabled=[true/false]", "internal-communication.shared-secret=<shared-secret-value>" and "internal-communication.jwt.expiration-seconds=<value in seconds>".
```

Comment:
Let's add a link to the JWT documentation.

Reply:
@nmahadevuni Please update.

Reply:
@czentgr Can you update the Prestissimo doc ?

Reply:
The option is documented at
`../develop/presto-native.rst`. This contains a list of options for prestissimo. 
The actual link with anchor should be
`../develop/presto-native.html#jwt-authentication-support`.

Do I need to do anything else?

Reply:
That should be it.